x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ManageOwnApiKeyClusterPrivilege.java

```diff
@@ -62,7 +63,7 @@ private ManageOwnClusterPermissionCheck() {
         protected boolean extendedCheck(String action, TransportRequest request, Authentication authentication) {
             if (request instanceof CreateApiKeyRequest) {
                 return true;
-            } else if (request instanceof UpdateApiKeyRequest) {
+            } else if (request instanceof UpdateApiKeyRequest || request instanceof BulkUpdateApiKeyRequest) {
```

Comment:
I'm going for an explicit list of classes instead of using `BaseUpdateApiKeyRequest` here because it's less risky to omit a potential future sub-class than to accidentally include it. One is a functional failure, the other a harder to detect potential security issue.